frontend/src/layout/navigation-3000/sidepanel/panels/activation/SidePanelActivation.tsx

```diff
@@ -0,0 +1,123 @@
+import { LemonButton, LemonButtonWithSideActionProps } from '@posthog/lemon-ui'
+import { useActions, useValues } from 'kea'
+import { ProfessorHog } from 'lib/components/hedgehogs'
+import { IconCheckmark, IconClose, LemonIconProps } from 'lib/lemon-ui/icons'
+import { LemonProgressCircle } from 'lib/lemon-ui/LemonProgressCircle'
+import { eventUsageLogic } from 'lib/utils/eventUsageLogic'
+
+import {
+    activationLogic,
+    ActivationTaskType,
+} from '~/layout/navigation-3000/sidepanel/panels/activation/activationLogic'
+
+import { SidePanelPaneHeader } from '../../components/SidePanelPaneHeader'
+
+export const SidePanelActivation = (): JSX.Element => {
+    const { activeTasks, completionPercent, completedTasks } = useValues(activationLogic)
+
+    return (
+        <>
+            <SidePanelPaneHeader title="Quick start" />
```

Comment:
Diff here is annoyingly not picked up - main addition is this header and moving the content of "ActivationTask" here.